### PR TITLE
#15: eslint configuration for react-tsx

### PR DESCRIPTION
**Pull request by deveth0**
Currently the react-tsx example relies on the eslint provided by the root folder. We should add an example, how the eslint config would work here to give a real stand-alone example-project.

This might especially be interesting, as eslint does not work with typescript out of the box and a complete example might help here.

### CODE REVIEW

**Comment by deveth0** [comment]
I tried to figure out, how we could add / support eslint for the react-tsx example project. 

Please have a look into this.

One open thing: when I print out the effective configuration (`/node_modules/.bin/eslint --print-config src/js/app.tsx | grep -a5 ecmaFeatures`), it still uses "ecmaVersion": 2017. Should we update this?

File: examples/react-tsx/.eslintignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+**/node_modules/**
+**/bin/**
+**/lib/**
```
Comment (friewerts) [inline examples/react-tsx/.eslintignore:3]: there is no lib (and bin) folder in your example. So you don't need to exclude them.

File: examples/react-tsx/src/components/Hello.tsx [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import React from "react";
 
-interface HelloProps { compiler: string; framework: string; }
+interface HelloProps { compiler: string; framework: string }
```
Comment (friewerts) [inline examples/react-tsx/src/components/Hello.tsx]: is this correct? looks wrong

File: examples/react-tsx/.eslintignore (hunk 1 above)
Comment (deveth0) [reply examples/react-tsx/.eslintignore:3]: done :)

**Comment by friewerts** [comment]
can we merge this one @deveth0 ?

**Comment by deveth0** [comment]
Yeah sure!

**Comment by deveth0** [comment]
Sry, wrong button :)

**Pull request merged**